community/cargo-static/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: Rasmus Thomsen <oss@cogitri.dev>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+pkgname=cargo-static
+pkgver=0.32.0
+pkgrel=1
+pkgdesc="Rust package manager - static version for bootstrapping"
+url="http://www.rust-lang.org"
+arch="x86_64"
+license="Apache-2.0 MIT"
+depends="rust"
+makedepends="
+	curl-dev
+	openssl-dev
+	cargo-bootstrap
+	"
+source="$pkgname-$pkgver.tar.gz::https://github.com/rust-lang/cargo/archive/${pkgver}.tar.gz
+	https://alpha.de.repo.voidlinux.org/distfiles/cargo-0.33.0-x86_64-unknown-linux-musl.tar.xz
```

Comment:
We will be able to remove this once this has been built once and instead use cargo-bootstrap which is provided by this package.